extensions/amp-call-tracking/amp-call-tracking.md

```diff
@@ -32,7 +32,7 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
-    <td>fill, fixed, fixed-height, flex-item, responsive</td>
+    <td>container, fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
```

Comment:
Hmm, okay, but what does the [validator](https://github.com/ampproject/amphtml/blob/master/extensions/amp-call-tracking/0.1/validator-amp-call-tracking.protoascii#L73) for amp-call-tracking have more layouts?

```
    supported_layouts: CONTAINER
    supported_layouts: FILL
    supported_layouts: FIXED
    supported_layouts: FIXED_HEIGHT
    supported_layouts: FLEX_ITEM
    supported_layouts: NODISPLAY
    supported_layouts: RESPONSIVE
```